src/ppl/src/IOPlacer.cpp

```diff
@@ -1775,9 +1808,13 @@ void IOPlacer::initConstraints(bool annealing)
         logger_->warn(PPL,
                       110,
                       "Constraint has {} pins, but only {} available slots.\n"
-                      "Increase the region in at least {}um.",
+                      "Increase the region {}-{} on the {} edge in at least "
+                      "{}um.",
```

Comment:
to at least

Reply:
"to at least" would imply that the message would inform the new total length of the region, right? The way it is currently written informs how much the user needs to increase the region.

Reply:
from ... to ...